tox.ini

```diff
@@ -25,3 +25,22 @@ basepython=python2
 
 [testenv:py3]
 basepython=python3
+
+[lint]
+skip_install=true
+commands=
+    pylint --rcfile=pylintrc rebasehelper
+deps=
+    pylint>=1.4.5
```

Comment:
How about creating `lint-requirements.txt` file and reference it here and also include it in `test-requirements.txt`?

Reply:
That is a great idea. The duplication code "pylint>=1.4.5" was my concern.
Let me try it.